README.md

```diff
@@ -71,6 +71,22 @@ Overrides the pages index generation with the passed jsonl pages.
 wacz create tests/fixtures/example-collection.warc -p passed_pages.jsonl
 ```
 
+### -e --extra-pages
+
+Overrides the extra pages index generation with the passed extra jsonl pages.
+
+```
+wacz create tests/fixtures/example-collection.warc -p passed_pages.jsonl -e extra_pages.jsonl
+```
+
+### -c --copy-pages
+
+Overrides the behavior of --pages and --extra-pages options to copy existing pages.jsonl and/or extraPages.jsonl as-is directly into the WACZ rather than parsing their contents.
```

Comment:
```suggestion
Overrides the behavior of --pages and --extra-pages options to copy existing pages.jsonl and/or extraPages.jsonl as-is directly into the WACZ rather than attempting to match each page to WARC record. The files are still parsed for basic correctness
```